### PR TITLE
Adding documentation for disabling compression in client-go

### PR DESCRIPTION
**Pull request by ckuduvalli**
Signed-off-by: Chaitanya Kuduvalli Ramachandra <chaitanya.r@ETC02Y77ZVJG5H.local>

Adding a parameter to disable compression in client-go
https://github.com/kedacore/keda/issues/3863

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.9/operate/cluster.md [hunk 1]
```diff
@@ -90,10 +90,11 @@ Some scalers issue HTTP requests to external servers (i.e. cloud services). As c
...
 
-| Adapter Flag   | Client Config Setting   | Default Value | Description                                                    |
-| -------------- | ----------------------- | ------------- | -------------------------------------------------------------- |
-| kube-api-qps   | cfg.QPS                 | 20.0          | Set the QPS rate for throttling requests sent to the apiserver |
-| kube-api-burst | cfg.Burst               | 30            | Set the burst for throttling requests sent to the apiserver    |
+|     Adapter Flag    | Client Config Setting   | Default Value | Description                                                    |
+| ------------------- | ----------------------- | ------------- | -------------------------------------------------------------- |
+| kube-api-qps        | cfg.QPS                 | 20.0          | Set the QPS rate for throttling requests sent to the apiserver |
+| kube-api-burst      | cfg.Burst               | 30            | Set the burst for throttling requests sent to the apiserver    |
+| disable-compression | cfg.DisableCompression  | false         | Disable compression for response in k8s restAPI in client-go   |
```
Comment (zroubalik) [inline content/docs/2.9/operate/cluster.md:97]: We should probably reference this kuberentes issue in the description: https://github.com/kubernetes/kubernetes/issues/112296

```yaml
Disable compression for response in k8s restAPI in client-go, see [this Kubernetes issue](https://github.com/kubernetes/kubernetes/issues/112296) for details
```

**Review by zroubalik — CHANGES_REQUESTED** [review]

File: content/docs/2.9/operate/cluster.md (hunk 1 above)
Comment (ckuduvalli) [reply content/docs/2.9/operate/cluster.md]: done please check

**Review by zroubalik — APPROVED** [review]

**Review by zroubalik — APPROVED** [review]

**Comment by ckuduvalli** [comment]
> Default value is true after last change

done

**Review by JorTurFer — APPROVED** [review]
Thanks! ❤️

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**